src/Csv.php

```diff
@@ -332,13 +332,35 @@ class Csv {
      * Constructor
      * Class constructor
      *
-     * @param  string|null  $input          The CSV string or a direct filepath
-     * @param  integer|null $offset         Number of rows to ignore from the beginning of  the data
-     * @param  integer|null $limit          Limits the number of returned rows to specified amount
-     * @param  string|null  $conditions     Basic SQL-like conditions for row matching
-     * @param  null|true    $keep_file_data Keep raw file data in memory after successful parsing (useful for debugging)
+     * @param  string|null $input The CSV string or a direct filepath
+     * @param  integer|null $offset Number of rows to ignore from the beginning
+     *   of  the data
+     * @param  integer|null $limit Limits the number of returned rows to
+     *   specified amount
+     * @param  string|null $conditions Basic SQL-like conditions for row
+     *   matching
+     * @param  null|true $keep_file_data Keep raw file data in memory after
+     *   successful parsing (useful for debugging)
      */
-    public function __construct($input = null, $offset = null, $limit = null, $conditions = null, $keep_file_data = null) {
+    public function __construct($input = NULL, $offset = NULL, $limit = NULL, $conditions = NULL, $keep_file_data = NULL) {
```

Comment:
Groß- und Kleinschreibung: Von Drupal bin ich `NULL` gewöhnt, von RDO und diesem Projekt `null`. Hast du es mit Absicht geändert? Gibt es einen Standard? 

Mir ist es ziemlich gleich welche Variante, doch ist Konsistenz innerhalb des Projektes wichtig.

Reply:
Sorry, I think that comes from our project. I think standard is lowercase and I prefer this too. Will change it.

Reply:
done